Dockerfile-django

```
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /api
+
+COPY requirements.txt /api/
```

change this to be able to launch the application
```suggestion
COPY . ./

RUN apt-get update && apt-get install -y libpq-dev build-essential

RUN pip install --no-cache-dir -r requirements.txt
```